test/extended/testdata/cmd/hack/lib/cmd.sh

```diff
@@ -210,7 +210,7 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	local test_succeeded=$( ${test_eval_func} "${test_result}"; echo $? )
 
 	local end_time=$(os::cmd::internal::seconds_since_epoch)
-	local time_elapsed=$(echo "scale=3; ${end_time} - ${start_time}" | bc | xargs printf '%5.3f') # in decimal seconds, we need leading zeroes for parsing later
+	local time_elapsed=$(echo $(( ${end_time} - ${start_time} )) | xargs printf '%5.3f') # in decimal seconds, we need leading zeroes for parsing later
```

Comment:
why has this changed?

Reply:
The `cli` image does not contain the `bc` binary `yum install -y bc` wouldn't work because it required subscription, this was the easiest way for me to work around that.